Bundle/QueryBundle/Helper/QueryHelper.php

```diff
@@ -142,6 +142,11 @@ public function buildWithSubQuery(VictoireQueryInterface $containerEntity, Query
         //Add ORDER BY if set
         if ($orderBy = json_decode($containerEntity->getOrderBy(), true)) {
             foreach ($orderBy as $addOrderBy) {
+                // Ignore this order if one parameters is missing.
+                if ((!isset($addOrderBy['by'])) || (!isset($addOrderBy['order']))) {
+                    continue;
```

Comment:
@paulandrieux Can you please give me an hint about the Exception that can be thrown here? Thanks.